monitoring/install.sh

```diff
@@ -15,20 +15,96 @@
 # limitations under the License.
 #
 
+
+usage() {
+    {
+        echo "This script installs the entire monitoring stack"
+        echo ""
+        echo "Usage:"
+        echo "$0 [-h] [-d] [-r] [-k <aws_access_key>] [-s <aws_secret_key>] [-n <namespace>]"
+        echo ""
+        echo "-d    Dry-run. Print what would be done without executing"
+        echo "-h    Show this help message"
+        echo "-k    AWS access key"
+        echo "-n    The namespace in which to install the monitoring stack. Defaults to 'monitoring'"
+        echo "-r    Enable prometheus remote write (AWS). Requires that -k and -s are set"
+        echo "-s    AWS secret key"
+    } 1>&2
+
+    exit 1
+}
+
+# TODO: Is it secure to pass the secret key via the command line? Or should we pass this via an env var?
```

Comment:
Should we pass this via the command line? If not I will use env vars for the AWS credentials.

Reply:
I think both work for me. Passing by environment variables is more secure in some cases (command line arguments can be read with `ps -ef`), but since it is just a quick script so IMO it's fine to pass as arguments.